test/test_dynamic_generator.jl

```diff
@@ -482,6 +482,7 @@ end
     @test dynamics[1] == Gen1AVR
     @test get_dynamic_injector(static_gen) == Gen1AVR
     @test get_base_power(static_gen) == get_base_power(Gen1AVR)
+    @test PSY.compare_values(static_gen, deepcopy(static_gen))
```

Comment:
I don't object, but out of curiosity, what is this for?

Reply:
oh, I added this in a different PR. The Git history is a bit messed up, this should be merged to `psy5` separately so it doesn't appear in this diff.